qiita_db/processing_job.py

```diff
@@ -781,14 +784,22 @@ def _set_status(self, value):
             new_status = qdb.util.convert_to_id(
                 value, "processing_job_status")
 
-            if self.user.info['receive_processing_job_emails']:
-                # skip if software is internal
-                ignore_software = ('Qiita')
-                if self.command.software.name not in ignore_software:
-                    subject = ('Job status change: %s (%s)' % (
-                        self.command.name, self.id))
-                    message = ('New status: %s' % (new_status))
-                    qdb.util.send_email(self.user.email, subject, message)
+            if value not in ('waiting'):
+                if self.user.info['receive_processing_job_emails']:
+                    # skip if software is artifact definition
+                    ignore_software = ('artifact definition', )
+                    if self.command.software.name not in ignore_software:
+                        ignore_commands = ('Validate', 'complete_job',
+                                           'release_validators')
+                        if self.command.name not in ignore_commands:
+                            subject = 'Job status change: %s (%s)' % (
```

Comment:
I like to use %s myself when I have the choice. Just want to confirm that it's okay to use %s and not f'{}' in new code.

Reply:
Agreed this is a matter of preference and what works well for placating flake8.